compiler/util-io/src/org/jetbrains/kotlin/konan/file/File.kt

```diff
@@ -56,6 +56,9 @@ data class File constructor(internal val javaPath: Path) {
     val listFilesOrEmpty: List<File>
         get() = if (exists) listFiles else emptyList()
 
+	val fileKey: Any
```

Comment:
Please add a brief comment explaining what is a file key and how it should be used.

Reply:
Updated with comment, and added a little bullet proofing if filekey is not actually available.

Reply:
Soft ping. Can you let me know if this works for you?

Reply:
Soft ping again.